include/libtorrent/torrent_handle.hpp

```diff
@@ -840,7 +840,7 @@ namespace libtorrent
 		// already exist your torrent will be set on top of it and remaining once
 		// get posion + 1. If the given position is greater then queue it will be
 		// added on bottom
```

Comment:
Sorry, I forgott your commet "updates the position in the queue for this torrent. The relative order of all other torrents remain intact but their numerical queue position shifts to make space for this torrent's new position"